recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -326,6 +329,11 @@ The next steps are:
             set_onboarded accountdevice
         fi
         echo "response: ${response}"
+        echo "After the successful onboarding the awsclient needs to be restarted 3 times."
```

Comment:
Rephrasing: "After a successful onboarding the awsclient needs to be rerun with 'systemctl restart awsclient' three times."